VideoLocker/src/main/java/org/edx/mobile/task/GetFollowingThreadListTask.java

```diff
@@ -41,7 +41,7 @@ public TopicThreads call( ) throws Exception{
                 String order;
                 if (orderBy == DiscussionPostsSort.LAST_ACTIVITY_AT) order = "last_activity_at";
                 else if (orderBy == DiscussionPostsSort.VOTE_COUNT) order = "vote_count";
-                else order = "";
+                else order = "comment_count";
```

Comment:
This key should ideally be a property of the `DiscussionPostsSort` enum, just as the display text resource IDs are. If the property is set up as a `public final` variable, then it can be referenced directly without a get method, but the implementation of it is up to you.